fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/dao/ReindexResourceDAO.java

```diff
@@ -51,6 +51,13 @@
     // Note that currently the global logical_resources table does not carry
```

Comment:
The is_deleted flag is now available on the LOGICAL_RESOURCES table.

Reply:
Updated.